src/mcts/stoppers/alphazero.cc

```diff
@@ -36,33 +36,49 @@ class AlphazeroTimeManager : public TimeManager {
   AlphazeroTimeManager(int64_t move_overhead, const OptionsDict& params)
       : move_overhead_(move_overhead),
         alphazerotimepct_(
-            params.GetOrDefault<float>("alphazero-time-pct", 12.0f)) {
+            params.GetOrDefault<float>("alphazero-time-pct", 12.0f)),
+        alphazeroincrementpct_(
+            params.GetOrDefault<float>("alphazero-increment-pct", 95.0f)) {
     if (alphazerotimepct_ < 0.0f || alphazerotimepct_ > 100.0f)
       throw Exception("alphazero-time-pct value to be in range [0.0, 100.0]");
+    if (alphazeroincrementpct_ < 0.0f || alphazeroincrementpct_ > 100.0f)
```

Comment:
There should be `{` `}` around the body of the if statement if it doesn't fit one line. (also in the line above)